pages/Seasonal-Categories.md

```diff
@@ -0,0 +1,9 @@
+# Seasonal Categories
```

Comment:
Wiki pages should not include headers.
They have headers automatically, this would create two.